lib/puppet/type/vrf.rb

```diff
@@ -0,0 +1,86 @@
+require_relative '../../puppet_x/puppetlabs/netdev_stdlib/check'
+if PuppetX::NetdevStdlib::Check.use_old_netdev_type
+  Puppet::Type.newtype(:syslog_settings) do
```

Comment:
There is no need to support legacy modules for this type. Please remove everything except the rsapi type here.

Create a temporary new module and use `pdk new provider` there to get a skeleton type and unit tests for this.